coderd/database/migrations/000091_add_workspace_agent_state.up.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TYPE workspace_agent_lifecycle_state AS ENUM ('created', 'starting', 'start_timeout', 'start_error', 'ready');
+
+ALTER TABLE workspace_agents ADD COLUMN lifecycle_state workspace_agent_lifecycle_state NOT NULL DEFAULT 'created';
```

Comment:
nit: Does it mean that *all* existing agents will be marked as created? Do they need to be restarted to align the real status?

Reply:
Great catch, I'll change default value to `ready` when adding the column, and then change it to `created` after the column is added 👍🏻.

Reply:
Fixed!